### DISCUSSION TITLE
Add option to flip PDFs for upsidedown scans in the Administration UI

### DISCUSSION THREAD

**Discussion by fwiessner** [jonaswinkler/paperless-ng#800] [Feature Requests]
Sometimes documents are scanned upside down - add ability to flip them or detect upside down documents and change orientation.

**Comment by jonaswinkler** [comment]
Paperless already has this. See PAPERLESS_OCR_ROTATE_PAGES in the documentation.

**Reply by fwiessner** [reply]
Yes, it would be cool if that would be configureable through the admin section via the webserver.

**Reply by fwiessner** [reply]
it also would be cool to do that manually if paperless failed somehow rotating the pages correctly
